src/app/pages/applications/forms/kubernetes-settings.component.ts

```diff
@@ -106,24 +98,13 @@ export class KubernetesSettingsComponent {
       }
     })
     const v4_interface_control = _.find(this.fieldSets[1].config, {'name' : 'route_v4_interface'});
-    const v6_interface_control = _.find(this.fieldSets[1].config, {'name' : 'route_v6_interface'});
     this.appService.getInterfaces().subscribe(interfaces => {
       interfaces.forEach(i => {
         v4_interface_control.options.push({ label: i.name, value: i.name });
-        v6_interface_control.options.push({ label: i.name, value: i.name });
       })
     })
-
   }
 
   beforeSubmit(data) {
-    if (data.route_v4_gateway === '') {
```

Comment:
Did we had to remove this for route v4? Or just wasnt necessary?

Reply:
I am not sure how this works, but just to be clear, we need to send `null` to allow user to unset this field.